src/EditorControls.js

```diff
@@ -298,6 +301,7 @@ EditorControls.propTypes = {
   plotly: PropTypes.object,
   showFieldTooltips: PropTypes.bool,
   traceTypesConfig: PropTypes.object,
+  useGLDefault: PropTypes.bool,
```

Comment:
I would have this be a full object, called `defaultInitialTrace` or something, so that we could have `bar` by default if we wanted.